### ISSUE TITLE
recommendation tweets (For you)

### ISSUE THREAD

**Issue by yazeed-Alshorman** [twitter/the-algorithm#310]
The 'For you' recommendations should be more based on user interests that are added to the profile, rather than the current algorithm based on accounts followed. It is not necessary that those followed have the same interests as me. The 'Following' section should display tweets, likes, and replies from those followed.

**Comment by Richmond-Nunoo** [comment]
Yeah this is a great suggestion!!Tiktok algorithm is designed this way that's why it's users is growing exponentially!!

**Comment by aloiscochard** [comment]
Closed as duplicate of https://github.com/twitter/the-algorithm/issues/1760.
